.github/workflows/python-package.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
```

Comment:
It's nice to have the tests run when you make pull requests to branches other than master also :slightly_smiling_face: